samplers/jaegerremote/go.mod

```diff
@@ -0,0 +1,15 @@
+module go.opentelemetry.io/contrib/samplers/jaegerremote
+
+go 1.16
+
+require (
+	github.com/gogo/protobuf v1.3.2
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/otel v1.4.1
+	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.4.1 // indirect
```

Comment:
This doesn't seem correct.  This module should not be depending on any exporter, even indirectly.

Reply:
Thank you. Please wait a few minutes and I'll submit it.

Reply:
done .
@Aneurysm9

Reply:
@yurishkuro @Aneurysm9 

Teacher, please help run CI again. Fixed the problem caused by Jaeger agent's non default value.